week-1/implement/sum.js

```diff
@@ -0,0 +1,9 @@
+function sum(arr){
+    const nonNumerical = arr.filter(item => typeof item === "number" && !isNaN(item));
```

Comment:
It's actually numerical in here

Reply:
yes this filters out only numerical value from array

Reply:
Yes right I fix this its important to write the variable names correctly thanks